go/service/contacts.go

```diff
@@ -20,11 +20,18 @@ type bulkLookupContactsProvider struct{}
 
 var _ contacts.ContactsProvider = (*bulkLookupContactsProvider)(nil)
 
+func (c *bulkLookupContactsProvider) LookupAllWithToken(mctx libkb.MetaContext, emails []keybase1.EmailAddress,
+	numbers []keybase1.RawPhoneNumber, userRegion keybase1.RegionCode, token contacts.Token) (contacts.ContactLookupResults, error) {
+	defer mctx.TraceTimed(fmt.Sprintf("bulkLookupContactsProvider#LookupAllWithToken(len=%d)", len(emails)+len(numbers)),
+		func() error { return nil })()
+	return contacts.BulkLookupContacts(mctx, emails, numbers, userRegion, token)
```

Comment:
I think the tokens are implementation-specific enough that we can just handle them inside of `contacts.BulkLookupContacts` and not "leak" outside of that package. What do you think?

Reply:
hm, that can't be exactly that... maybe that's necessary after all, too bad

Reply:
yeah i tried to make it not leak but found it hard with them both satisfying the same interface